creator-node/src/services/stateMachineManager/stateMonitoring/types.ts

```diff
@@ -81,5 +69,5 @@ export type FindReplicaSetUpdateJobParams = {
   parentSpanContext?: SpanContext
 }
 export type FindReplicaSetUpdatesJobReturnValue = {
-  cNodeEndpointToSpIdMap: CNodeEndpointToSpIdMap
+  cNodeEndpointToSpIdMap: string
```

Comment:
ah bc its gotta be serialized for bull?

Reply:
yeah just cause it's not an object that we can enforce types on anymore -- it's a Map and just returning the Map won't print it properly on the dashboard